src/gengowatcher/config.py

```
@@ -235,7 +248,9 @@ def load_config(self):
                     try:
                         with open(self.CONFIG_FILE, "w", encoding="utf-8") as f:
                             self._config_parser.write(f)
-                        print(f"Config file updated with missing sections/options")
+                        print(
+                            f"WARNING: Config file updated with missing sections/options"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)